contents/handbook/company/docs.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Documentation
+sidebar: Handbook
+showTitle: true
+---
+
+## Responsibilities
+
+- **Product teams** are responsible for ensuring their docs are up-to-date. This means:
+  - Documenting new features when they're launched.
+  - Correcting mistakes reported by users.
+  - Clarifying documentation based on user feedback and support tickets.
+
+- **Content marketing** is responsible for improving the docs. This means:
+  - Reviewing draft documentation created by product teams.
+  - Identifying and improving low quality documentation.
+  - Ensuring screenshots and other visual elements are up-to-date.
+  - Shipping supplementary docs and tutorials based on feedback and emerging use cases.
+
+- **Website & Docs** is responsible for design, organization, and discovery. This means:
```

Comment:
I guess it's safe to rename Website & Docs to just Website if the content of the docs are out of their purview